version.txt

```diff
@@ -1,4 +1,4 @@
-FLATCAR_VERSION=2345.0.0
-FLATCAR_VERSION_ID=2345.0.0
+FLATCAR_VERSION=2430.0.0
+FLATCAR_VERSION_ID=2430.0.0
 FLATCAR_BUILD_ID=""
-FLATCAR_SDK_VERSION=2331.0.0
+FLATCAR_SDK_VERSION=2411.0.0
```

Comment:
`flatcar-master` is for stable and beta, so why is the SDK of the alpha release 2411.0.0 used to build it?

Reply:
Good question.
So far we do not distinguish alpha/edge from stable/beta, w.r.t. branch names in the manifest repo. We only have one single branch `flatcar-master`.
We can set the version to 2345.0.0 for this PR.
Anyway in the next few weeks, we would have to set it to 2411.0.0.

Reply:
I would like it to be the stable version because by default the repositories check out `flatcar-master`.